src/main/resources/templates/provider/list.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head">
```

Comment:
It would be worth using the `fragments.html` segment that already shows navigation links.

Reply:
I also tried adding the bootstrap elements from the fragments page, but no change in appearance took place